src/components/Dropdown.js

```diff
@@ -0,0 +1,300 @@
+import React from 'react'
+import ReactSelect from 'react-select'
+import CreatableSelect from 'react-select/creatable'
+import AsyncSelect from 'react-select/async'
+import AsyncCreatableSelect from 'react-select/async-creatable'
+import styled, { withTheme, css } from 'styled-components'
+import { components } from 'react-select'
+import customCursor from '../assets/custom-cursor.png'
+
+const dropdownWidth = {
+  normal: '300px',
+  searchable: '400px',
+}
+
+const sharedStyle = css`
+  .react-select__control {
+    background-color: transparent;
+    margin: 2em 0;
```

Comment:
I took another look - can we decrease the top margin and increase the bottom? Also you can refer to my other comment below for the additional bottom margin
```suggestion
    margin: 1em 0 2.5em 0;
```